README.md

```diff
@@ -1 +1,65 @@
 # PCBs
+
+## About
+Central repository of all PCB design files. Includes:
+
+| Folder    | Project Name                     |
+|-----------|----------------------------------|
+| bspd      | Brake System Plausibility Device |
+| hv-sense  | High Voltage Sense               |
+| latching  | Latch Circuit                    |
+| precharge | Precharge Circuit                |
+| tsal      | Tractive System Active Light     |
+| vcu       | Vehicle Control Unit             |
+
+## Version Control for KiCad Projects
+
+### Why?
+- Revert individual changes.
+- Document development history via commits and GitHub issues.
+- Easily view previous versions of designs.
+- Manage shared library of components.
+
+### Files
+
+Files that should be in this repo, under version control:
+- Project files (`.kicad_pro`).
+- Schematic files (`.kicad_sch`).
+- PCB files (`.kicad_pcb`).
+- Custom footprints (`.kicad_mod`).
+- Custom symbols (`.kicad_sym`).
+- Footprint library tables (`fp-lib-table`).
+- Symbol library tables (`sym-lib-table`).
+- 3D models of components (`.step` files).
+- Simulation code (e.g. `.asc` files from LTSpice).
+
+Files that should never be in here:
+- Exports such as BOMs, gerbers, netlists, renders, etc.
```

Comment:
I think we might want to include a lot of these things. There should be a standard format defined for what these files should be and where they should live within each PCB project, but having gerbers, schematic PDFs, PCB drawings, BOM lists on hand when you open an project is probably a good thing.

We would need to encourage people to keep these up to date, but they should really only be created at the end of the project anyway, and only require minor alterations from there.

It might be something to discuss further...

Reply:
My thought was that because these are created at the end of a project and they are exports from the design rather than design files, they wouldn't be kept under version control. In my head the analogy is that those exports are kind of like when you build source code into an executable - yes you want to have the exe on hand when you reach a point of releasing it, but the release itself wouldn't be kept under version control like the source code. My worry is that the repo is going to get very large over time with those kinds of files and will start to become messy and slow to clone.

As a possible solution, what do you think about having those files as GitHub releases?  Once a design is completed, we can make a release with the BOM, gerbers, etc and then that will be associated with a specific tagged commit and can easily be downloaded. That keeps a good link between the design files and the exports, without bloating the repo.

Reply:
I guess the release might work - they'd just have to be something to determine when a new release is warranted. I think they'd also be a few questions to answer like what if there was a small minor update to a schematic after a release, does that count as a new release etc...

Might be something to discuss further at some point

Reply:
I'm gonna merge this just so we have _some_ instructions. As discussed, we can speak further when we meet to go over the PCBs and figure something final out. None of these files should be being generated or added in the super near future, so we should have it sorted by the time that starts to happen.